Sources/SwiftUIPager/PagerContent.swift

```diff
@@ -252,10 +255,25 @@ extension Pager.PagerContent {
     var swipeGesture: some Gesture {
         DragGesture(minimumDistance: minimumDistance, coordinateSpace: .global)
             .onChanged({ value in
-                self.onDragChanged(with: value)
+                if forwardOnly {
+                    if isForwardGesture(value) {
```

Comment:
* I think this change would only be forward only, if you had forward and backward it wouldn't make sense as you can't go backwards on the first page.

* Just putting this up here for initial review, see what you think.

* One issue with this code is that it won't work for when dragging is disabled but maybe you know what change is needed for this

Reply:
> I think this change would only be forward only, if you had forward and backward it wouldn't make sense as you can't go backwards on the first page.
you're right, backwards makes no sense.
> Just putting this up here for initial review, see what you think.
👍
> One issue with this code is that it won't work for when dragging is disabled but maybe you know what change is needed for this 
do you mean when used with `disabledDragging`? I think that's expected, at the end this is forcing the user move in one direction. If dragging is disabled and the paging is programmatic, then it's up to the developer to decide.

Perhaps we can make it more explicit as in `.dragForwardOnly(_:)`? That would help understand that if dragging is disabled then this modifier serves no purpose

Reply:
updated to be dragForwardOnly